It was modificated folder structure.

Anya2ka opened the pull request:
This closes #12.

Anya2ka committed:
It was modificated folder structure.

This closes #12.

iviaks reviewed (changes requested):
@Anya2ka, Проверь свой код на работоспособность

Anya2ka committed:
Modified folders in project

This closes #12.
1. Base and bootstrap folders were created in repository;
2. Style folder was renamed in css;
3. Added js and img folders in base folder;
4. Styleposition file was renamed in position.
